source/conf.py

```
@@ -8,37 +8,47 @@
-copyright = '2025, Richard. A. Schaefer'
-author = 'Richard. A. Schaefer'
-release = '0.0.1'
+project = "genogrove"
+copyright = "2025, Richard. A. Schaefer"
+author = "Richard. A. Schaefer"
+release = "0.0.1"
 
 # -- General configuration ---------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
 
 import os, sys
```

Ruff (0.13.3)

Redefinition of unused `os` from line 2

Remove definition: `os`

(F811)